_posts/2024-09-15-labels-talk/labels-talk.Rmd

```diff
@@ -106,15 +103,27 @@ I also tried to make the title more compelling and to imbue more energy and enth
 
 ## Talk preparation
 
-| Time before conference | Organizer    | Task                                             |  
+Posit provided training by Articulation for all conference speakers, and it was a great forum to discuss ideas and get feedback.
+The coach led sessions were 2 hours each, in which ~5 presenters took turns sharing their talk ideas and feedback on others'. 
+Having goals at fixed timepoints helped to pace the work, and I incorporated both the coach's and fellow speakers  diverse feedback at each stage.
+
+| Time before conference | Organizer    | Meeting topic                                    |  
 | ---------------------- | ------------ | ------------------------------------------------ |  
 | 10 weeks               | Articulation | Discuss main takeaways                           |
 | 7 weeks                | Articulation | Review outline                                   |
-| 4 weeks                | Articulation | Present draft slides for a portion of talk       |
+| 4 weeks                | Articulation | Present draft slides for portion of talk         |
 | 2 weeks                | Articulation | Present 10 min of talk                           |
 | 7 days                 | Self         | Full practice talk                               |
 | 4 days                 | Self         | Follow-up to review suggested changes            |
 
+However, not all portions of the talk, nor the full talk, were practiced during Articulate, and so the full content had not yet been pressure tested.
```

Comment:
Articulation or Articulate? (Full disclosure: I never remember :joy: )

Reply:
Articulation! thank you!